include/megaapi.h

```diff
@@ -3930,7 +3930,7 @@ class MegaGlobalListener
         virtual void onNodesUpdate(MegaApi* api, MegaNodeList *nodes);
 
         /**
-         * @brief This function is called when the account has been updated (upgraded/downgraded)
+         * @brief This function is called when the account has been updated (confirmed/upgraded/downgraded)
```

Comment:
I've added the _confirmed/_ to the other callback, `MegaListener::onAccountUpdate()`